config/badger_config.py

```diff
@@ -180,7 +180,7 @@
 globalStartTime = 1607014800
 
 badger_config = DotMap(
-    prod_json="deploy-final.json",
+    prod_json="deploy-test-digg.json",
```

Comment:
Using the digg deploy json since it has all contracts badger & digg. Needed this so the badger system would deploy all digg logic contracts correctly etc.